src/modules/views/Public/PublicFolderView.jsx

```diff
@@ -181,62 +182,67 @@ const PublicFolderView = () => {
       <ModalManager />
       {isSharingBannerPluginDisplayed && <SharingBannerPlugin />}
       <Content className={isMobile ? '' : 'u-ml-1 u-pt-1'}>
```

Comment:
please take a deep look to Dropzone component and implementation elsewhere, <Content> is embedded into it :)